Celery RQ migration and concurrency (#691)
nautobot/nautobot: nautobot/docs/installation/services.md

WantedBy=multi-user.target
```

#### Migrating to Celery from RQ

> Prior to migrating, you need to determine of you are running any custom background tasks or plugins that still rely on the RQ worker. There are a few ways to do this. Two of them are:

Reviewer: ```suggestion
Prior to migrating, you need to determine whether you have any plugins installed that run custom background tasks that still rely on the RQ worker. There are a few ways to do this. Two of them are:
```

(END)
```

> If you cannot update your custom tasks with the change described above, you must run the [RQ worker concurrently with the Celery worker](#concurrent-celery-and-rq-nautobot-workers) until the plugin can be updated.

Reviewer: ```suggestion
If you are using plugins that use custom background tasks but have not yet made the change described above, you must run the [RQ worker concurrently with the Celery worker](#concurrent-celery-and-rq-nautobot-workers) until the plugin can be updated.
```

$ sudo systemctl enable --now nautobot nautobot-worker
```

If you are also running the RQ worker, repeat the above command for the RQ service:

```no-highlight
sudo systemctl enable --now nautobot-rq-worker
```

!!! tip

>     If you are running the concurrent RQ worker, you must remember to enable/check/restart the `nautobot-rq-worker` process as needed, oftentimes in place of the `nautobot-worker` process. 

Reviewer 2: ```suggestion
    If you are running the concurrent RQ worker, you must remember to enable/check/restart the `nautobot-rq-worker` process as needed, oftentimes in addition to the `nautobot-worker` process. 
```

Reviewer 3: Thanks for adding this tip! But it's not quite correct since for v1.1.0 and later the `nautobot-worker` (Celery) service must always be running for background services to operate, since the RQ worker will only be used to execute custom tasks that haven't been migrated.

Reviewer 2: not sure I follow @jathanism's comment here, I think we are in agreement.  The idea of the tip is to remind users any time they need to enable/check/restart the nautobot-worker (celery) service they probably also need to perform the same action with the nautobot-rq-worker (rq) service

Author: added Nick's suggested edit, which made sense
